### PR TITLE
Add site quality histograms for v4

### PR DESCRIPTION
**Pull request by mike-w-wilson**
This adds the site quality metric histograms for v4. It is essentially a copy and paste from v3. 

v4 exome metric histogram parameters have been discussed here:  https://atgu.slack.com/archives/CRA2TKTV0/p1698085670870479 

This function will work on genomes but we will need to confirm the v4 genome ranges still look ok using the v3 ranges and then copy the file if they do, or modify it and put it in the v4 location if they need updates. 

These histograms are displayed on the variant pages and the variants specific metric gets a line added by the browser 
Screenshot 2023-10-23 at 2 08 16 PM

Exome test run here:
https://console.cloud.google.com/dataproc/jobs/a97858c504b244e3a9bb37f5e6439549/monitoring?region=us-central1&project=broad-mpg-gnomad

### CODE REVIEW

**Review by KoalaQin — CHANGES_REQUESTED** [review]
A few comments, btw, if you already update  the json file for exomes, can you just copy one to the genomes path? I copied the v3 json but not the up-to-date ranges for both.

**Comment by mike-w-wilson** [comment]
Yes -- I'll handle copying to genomes and running this for both datatypes

File: gnomad_qc/v4/create_release/make_var_annot_hists.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                            hl.log10(ht.info.AS_QUALapprox),
+                            *ANNOTATIONS_HISTS["AS_QUALapprox"],
+                        ),
+                    )
+                ).map(lambda x: x[1].annotate(metric="AS_QUALapprox-" + x[0]))
+            ),
+            _localize=False,
+        )
+
+        # Defining hist range and bins for allele frequency groups because they
```
[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad_qc/v4/create_release/make_var_annot_hists.py:167]: I meant above this note, you got the hists for defined range and bins for all annotations but inbreeding coefficient, and below this note, you're doing this for inbreeding coefficient to get its hists.

File: gnomad_qc/v4/create_release/make_var_annot_hists.py [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+LOG10_ANNOTATIONS = ["AS_VarDP", "QUALapprox", "AS_QUALapprox"]
```
Comment (KoalaQin) [inline gnomad_qc/v4/create_release/make_var_annot_hists.py:26]: Another question here: you got 3 annotations as log10_annotations, but why from L135-L165, you only did log 10 for the last two of them?

Comment (KoalaQin) [reply gnomad_qc/v4/create_release/make_var_annot_hists.py:26]: Were you planning to get a histogram with their original values and with log10 scale?

Comment (mike-w-wilson) [reply gnomad_qc/v4/create_release/make_var_annot_hists.py:26]: All three get passed as log in L107. The last two also get split up by frequency bin.

Comment (KoalaQin) [reply gnomad_qc/v4/create_release/make_var_annot_hists.py:26]: Sorry I missed that. Could you add a small note before L135?

**Review by KoalaQin — APPROVED** [review]
LGTM! Thanks~

**Pull request merged**